### ISSUE TITLE
Build BertHead

### ISSUE THREAD

**Issue by pmhalvor** [pmhalvor/fgsa#5]
Checklist:
- [x] init BertModel object from pretrained
- [x] output layers for each task
- [x] optimizers for each task
- [x] predictions for each task
- [x] evaluation logs for each task
- [x] run long model over night

**Comment by pmhalvor** [comment]
![image](https://user-images.githubusercontent.com/31341520/155486793-6036db0a-a533-449d-a0c7-2468ca5ea7a5.png)

![image](https://user-images.githubusercontent.com/31341520/155487122-61dd6d8a-90ee-4581-ae97-05e8aa8a8060.png)

Some things to note about this model:
- Holder seems to be learning well, across batches
- Target and polarity end up syncing up around 20 epochs for both 75% and 50% of data, w/ some batch-wise fluctuations
    - Should probably have individual learning rate
- Expression shows very much batch-wise fluctuation, with little learning in the beginning
    - Could mean learning rate too high already from start
- Learning rate schedulers are made, but never turned on
    - First task today should be to learn how to activate these

**Comment by pmhalvor** [comment]
Another point to note is that no Linear layer has dropout here. To regularize a bit, a dropout can be provided in ìnit_components()`

**Comment by pmhalvor** [comment]
> Another point to note is that no Linear layer has dropout here. To regularize a bit, a dropout can be provided in `init_components()`

Linear layers don't have dropouts.
